backend/Domain/Models/NewUserModel.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+
+namespace OpenData.Domain.Models
+{
+    /// <summary>
+    /// Form for creating new user
+    /// </summary>
+    public class NewUserModel
```

Comment:
This as well looks like a Resource. The models folder should contain things that exist in databases, these classes seem to just be for validation

Reply:
Resolved in newer commit.